storage/mock/latestStorageDataProviderStub.go

```diff
@@ -0,0 +1,42 @@
+package mock
+
+import "github.com/ElrondNetwork/elrond-go/storage"
+
+// LatestStorageDataProviderStub --
```

Comment:
remove the second - (valid for all exported items)

Reply:
done.